src/main/java/ch/njol/skript/sections/SecWhile.java

```diff
@@ -20,6 +20,7 @@
 
 import ch.njol.skript.Skript;
 import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.doc.*;
```

Comment:
You should avoid * imports here

Reply:
> You should avoid * imports here

Why should wildcard imports be avoided? In case of conflicts, overriding the import is trivially done by adding an explicit import below the wildcard import, and if several classes are used, a wildcard import may drastically reduce the size of the somewhat useless header in the code (which is arguably already too long due to the copyright notice).

Reply:
I think the contributor style guidelines say don’t use it.

Reply:
When reading code on GitHub, and trying to find the package of a class with given simple name, it's much easier to find with only regular imports and no star imports